components/clp-py-utils/clp_py_utils/clp_config.py

```diff
@@ -777,6 +777,18 @@ def validate_logging_level(cls, field):
         return field
 
 
+class Presto(BaseModel):
+    @validator("host")
+    def validate_host(cls, field):
+        _validate_host(cls, field)
+        return field
+
+    @validator("port")
+    def validate_port(cls, field):
+        _validate_port(cls, field)
+        return field
```

Comment:
Similar to S3Credentials, we should still declare the field even if they don't have a default
```suggestion
class Presto(BaseModel):
    host: str
    port: int

    @validator("host")
    def validate_host(cls, field):
        _validate_host(cls, field)
        return field

    @validator("port")
    def validate_port(cls, field):
        _validate_port(cls, field)
        return field
```